lib/pacemaker/pcmk_output.c

```diff
@@ -518,7 +530,7 @@ locations_list(pcmk__output_t *out, va_list args) {
 
             PCMK__OUTPUT_LIST_HEADER(out, FALSE, rc, "Locations");
             out->list_item(out, NULL, "Node %s (score=%s, id=%s, rsc=%s)",
```

Comment:
I took some liberties when node is guaranteed to be non-NULL. It could still theoretically be "unidentified node" if both name and ID are NULL but that's not actually possible currently.

I wouldn't mind changing this instance except for backward compatibility. We do have XML output for crm_resource now but it hasn't been around long.